WorkflowTesting/Tests/WorkflowRenderTesterTests.swift

```diff
@@ -98,6 +98,19 @@ final class WorkflowRenderTesterTests: XCTestCase {
             .assertNoAction()
     }
 
+    func test_childWorkflowAction() {
```

Comment:
Would this test fail before? I thought this only happened when things got `AnyWorkflow`d?

Reply:
This test did fail before the fix.